### ISSUE TITLE
LeafletJS does not work on production server

### ISSUE THREAD

**Issue by SehrGuterCode** [Jugendhackt/wann-soll-ich-lueften.de#1]
LeafletJS does not work on the production web page on https://wann-soll-ich-lüften.de/, because the URL of the stylesheet and script is referenced with `http` instead of `https`. Browsers do not load resources from http urls if the web page uses https.
The map does therefore only work on localhost, because localhost uses http instead of https.

https://github.com/Jugendhackt/wann-soll-ich-lueften.de/blob/885abc090292f064abe6a96d0c628e65391ab1bc/server/templates/main.html#L58
https://github.com/Jugendhackt/wann-soll-ich-lueften.de/blob/885abc090292f064abe6a96d0c628e65391ab1bc/server/templates/main.html#L59

However, you're using an old CDN which does not support https. Use the new CDN from the LeafletJS docs:

https://leafletjs.com/download.html#using-a-hosted-version-of-leaflet

Other than that: Cool project, looking forward to how it will develop!

**Comment by LuisSchuimer** [comment]
Hi! Thank you for your hint! I also wondered why the card didn't work. But I haven't looked into it much. Unfortunately I don't know Java Script. I just took the code from the documentation. When I tested the version, everything worked. If you want I would ask you if you want to fix the bug! I wouldn't pay though as we don't make any money from the site. I would put you in the credits though. There were three of us when Hackt was a youth. I was responsible for the backend and not the frontend. So I'm very unquestioning about this bug.

Thanks!
Luis Schuimer
